### PR TITLE
Organization edit gallery

### PR DESCRIPTION
**Pull request by Omarkojak**
Sorry if the name conventions are bad :D

### CODE REVIEW

File: resources/views/event/gallery/index.blade.php [hunk 1]
```diff
@@ -25,6 +25,11 @@
                                 {!! method_field('DELETE') !!}
                                 <button type="submit" onclick="return confirm('Are you sure?');" class="btn btn-danger btn-event">Delete photo</button>
                             </form>
+                            <form action="{{ url('event/'.$event->id.'/editcaption/'.$photo->id) }}" method="POST">
+                                {!! csrf_field() !!}
+                                {!! method_field('POST') !!}
```
Comment (AhmadElsagheer) [inline resources/views/event/gallery/index.blade.php]: no need for this line

File: resources/views/event/gallery/index.blade.php [hunk 2]
```diff
@@ -25,6 +25,11 @@
...
+                                <button type="submit" class="btn btn-danger btn-event">Edit/add caption</button>
```
Comment (AhmadElsagheer) [inline resources/views/event/gallery/index.blade.php]: customize the message to be either add or edit

File: app/Http/routes.php [hunk 3]
```diff
@@ -250,6 +250,9 @@
     Route::post('event/{id}/gallery/upload','Event\EventGalleryController@upload');
     Route::post('event/{id}/gallery','Event\EventGalleryController@store');
     Route::delete('event/{id}/deletephoto/{photo_id}','Event\EventGalleryController@destroy');
+    Route::post('event/{id}/editcaption/{photo_id}','Event\EventGalleryController@editview');
```
Comment (AhmadElsagheer) [inline app/Http/routes.php]: get `event/{id}/photo/{photo_id}/edit`

File: app/Http/routes.php [hunk 4]
```diff
@@ -250,6 +250,9 @@
...
+    Route::post('event/{id}/savecaption/{photo_id}','Event\EventGalleryController@savecaption');
+
```
Comment (AhmadElsagheer) [inline app/Http/routes.php:255]: post `event/{id}/photo/{photo_id}/update`

File: resources/views/event/gallery/index.blade.php (hunk 1 above)
Comment (Omarkojak) [inline resources/views/event/gallery/index.blade.php]: anhy line m3lsh ?

Comment (AhmadElsagheer) [inline resources/views/event/gallery/index.blade.php]: method field

Comment (Omarkojak) [inline resources/views/event/gallery/index.blade.php]: tmam done :D

**Pull request merged**